modular_skyrat/modules/ashwalkers/code/effects/ash_rituals.dm

```diff
@@ -130,11 +137,12 @@
 
 /datum/ash_ritual/ash_ceremony/ritual_success(obj/effect/ash_rune/success_rune)
 	. = ..()
-	for(var/mob/living/carbon/human/human_target in range(1))
+	for(var/mob/living/carbon/human/human_target in range(2, get_turf(success_rune)))
```

Comment:
I'm not that experienced, but wouldn't `in range` check every atom for type? Wouldn't it be more reasonable to use `viewers()` for that?

Reply:
in range does check the atoms and the type-- but that is fine. Viewers are going to be specifically things that view the atom, which doesn't really fit what I want.